x-pack/test/api_integration/services/transform.ts

```diff
@@ -11,11 +11,13 @@ import { TransformAPIProvider } from '../../functional/services/transform/api';
 import { TransformSecurityCommonProvider } from '../../functional/services/transform/security_common';
 import { MachineLearningAPIProvider } from '../../functional/services/ml/api';
 import { MachineLearningTestResourcesProvider } from '../../functional/services/ml/test_resources';
+import { MachineLearningSecurityCommonProvider } from '../../functional/services/ml/security_common';
 
 export function TransformProvider(context: FtrProviderContext) {
   const api = TransformAPIProvider(context);
-  const mlApi = MachineLearningAPIProvider(context);
   const securityCommon = TransformSecurityCommonProvider(context);
+  const mlSecurityCommon = MachineLearningSecurityCommonProvider(context);
```

Comment:
Same as below, If `mlSecurityCommon` is not used in `MachineLearningSecurityCommonProvider`, we can revert changes in these two lines.

Reply:
https://github.com/elastic/kibana/pull/179321/commits/233592e50eb19d61a76fb9648f80293dbe366191